api/include/opentelemetry/trace/trace_state.h

```diff
@@ -97,37 +107,163 @@ class TraceState
     }
   };
 
-  // An empty TraceState.
-  TraceState() noexcept : entries_(new Entry[kMaxKeyValuePairs]), num_entries_(0) {}
+  /**
+   * Returns a newly created Tracestate parsed from the header provided.
+   * @param header Encoding of the tracestate header defined by
+   * the W3C Trace Context specification https://www.w3.org/TR/trace-context/
+   *  @return Tracestate A new Tracestate instance or DEFAULT
+   */
+  static TraceState FromHeader(const nostd::string_view &header)
```

Comment:
```suggestion
  static TraceState FromHeader(nostd::string_view header)
```
Just a nit, but to be consistent with the `Get` method below. `string_view` is const anyway, and it's also small enough to be passed by value.